vehicle_gateway_integration_test/test/CMakeLists.txt

```diff
@@ -22,16 +22,9 @@ add_launch_test(test_px4.py
   TIMEOUT 120
 )
 
-add_launch_test(test_px4.py
-  TARGET test_mc_fw_mc
-  ARGS "vehicle_type:=standard_vtol" "script_test:=mc_fw_mc.py"
-  TIMEOUT 120
-)
-
```

Comment:
This is in the `vehicle_gateway_python` package and didn't seem to be linking correctly.

Reply:
Umm build fails with this on top of Morgan's PR,
```
  file INSTALL cannot find
  "/usr/local/google/home/dharinidutia/vg/src/vehicle_gateway/vehicle_gateway_integration_test/test/script/mc_fw_mc.py":
  No such file or directory.
```